runtimes/polimec/src/xcm_config.rs

```diff
@@ -256,18 +251,24 @@ pub type Barrier = TrailingSetTopicAsId<
 >;
 
 /// Trusted reserve locations for reserve assets. For now we only trust the AssetHub parachain
-/// for the following assets: DOT, USDT, USDC.
+/// for the following assets: DOT, USDT and USDC.
 pub type Reserves = AssetHubAssetsAsReserve;
 
 /// Means for transacting assets on this chain.
 /// FungibleTransactor is a FungibleAdapter that allows for transacting PLMC.
 /// ForeignAssetsAdapter is a FungiblesAdapter that allows for transacting foreign assets.
 /// Currently we only support DOT, USDT and USDC.
 pub type AssetTransactors = (FungibleTransactor, ForeignAssetsAdapter);
+
 pub type TakeRevenueToTreasury =
 	cumulus_primitives_utility::XcmFeesTo32ByteAccount<AssetTransactors, AccountId, TreasuryAccount>;
+
+pub type PriceForParentDelivery = NoPriceForMessageDelivery<()>;
+
+pub type PriceForSiblingParachainDelivery = NoPriceForMessageDelivery<ParaId>;
```

Comment:
Please add comment to this that once we open channels with more parachains, we should probably update this. We allow no fee from AssetHub but might need to require fees from other chains